types/lazy/lazy.go

```diff
@@ -120,9 +120,9 @@ func (z *SyncValue[T]) PeekErr() (v T, err error, ok bool) {
 	return zero, nil, false
 }
 
-// TB is a subset of testing.TB that we use to set up test helpers.
+// testingꞏTB is a subset of testing.TB that we use to set up test helpers.
```

Comment:
Cute, but could we at least get something that doesn't look like a replacement rune? 😆

Reply:
This font gave me away. Crap. I thought it might slide through :) 

My first try was a boring ol' underscore. I'll revert back to that.